backend-models/service-tests/showDuggaservice-test.txt

```diff
@@ -0,0 +1,393 @@
+showDuggaservice.php
+
+—------------------------------------------------------------------------------------------------------------------------
+TEST #1 - 100-105
+Get active users
+
+***
+Pre-req: -
+***
+Send parameters inputs to showDuggaservice.php
+    send
+    {
+        $opt = “UPDATEAU”
+        $hash = hash
+
+    }
+
+    SQL-query: SELECT active_users FROM groupdugga WHERE hash=hash
+    This should show all active users
+
+***
+Gather service output
+***
+    Save the values of the echoed array as a JSON (at the end of the file. echo json_encode($array);)
+    This is the expected output for the micro service.
+
+—------------------------------------------------------------------------------------------------------------------
+TEST #2 - 106-110
+Create active users
+
+***
+Pre-req: $active == NULL
+***
+
+Send parameters inputs to showDuggaservice.php
+    send
+    {
+        $opt = “UPDATEAU”
+        $hash = “tj7dh2nb”
+        $AUtoken = 999
+    }
+
+    SQL-query: INSERT INTO groupdugga(hash,active_users) VALUES(:hash,:AUtoken)
+***
+* Get newly added active users
+***
+	SQL-query: SELECT MAX(active_users) FROM groupdugga; // Save to variable $active_users this will identify the last active user that was created (it has the highest AUtoken)
```

Comment:
a nitpick but should't
![image](https://user-images.githubusercontent.com/102600391/234591847-bf6b71a2-0f16-4dc4-9330-c9c497d836f3.png)
be active_user?

Reply:
active_users :p

Reply:
hmm.. i just realized something while checking the database. the only key value in the table groupdugga is hash. 
and if i have understood it correctly these values are random?

seeing as no value is auto incremented on creation, SELECT MAX(active_users) or SELECT MAX(hash) for that mater should not result in the latest user. 

my suggestion in that case is to save both sent values into a variable instead (active_users and hash)

Reply:
Instead of a select query for the max active users we now have a select for the hash instead